lib/spree/shipment_notice.rb

```diff
@@ -42,7 +42,7 @@ def capture_payments!
     end
 
     def process_payments!(order)
-      uncaptured_payments = order.payments.select(&:pending)
+      uncaptured_payments = order.payments.pending
       uncaptured_payments.each(&:capture!)
```

Comment:
Can we leave this as a one-liner? i.e: `order.payments.pending.each(&:capture!)`